ckb-bin/src/lib.rs

```diff
@@ -47,8 +47,9 @@ pub fn run_app(version: Version) -> Result<(), ExitCode> {
         }
     }
```

Comment:
Here it can be refactor like this, I think the intention of this change in clap is this
```rust
if let Some((cli, matches)) = app_matches.subcommand() {
     match cli {
           cli::CMD_INIT =>  {
                  ....
           },
     }
}
```

Reply:
refactor it, done, Thanks.